debug.log

```diff
@@ -0,0 +1,3 @@
+[1201/160449.729:ERROR:directory_reader_win.cc(43)] FindFirstFile: System nie mo�e odnale�� okre�lonej �cie�ki. (0x3)
```

Comment:
what is going on here?
Btw. did browser start?

Reply:
It's something from chrome when i shut down laptop, I will add it to gitignore.